src/darby.js

```diff
@@ -240,6 +251,74 @@ function respondToStartOddsEvent(event) {
   });
 }
 
+function respondToOddsPlayEvent(event) {
+  const userId = event.user;
+  const oddsPlayValue = event.text.match(INT_CHECK_REGEX)[0];
+
+  darbyDb.getOldestPendingOddsRecordForUser(userId, record => {
+    if (record === null) {
+      slackAction.sendMessage(
```

Comment:
No message here, IMO. If they're not playing odds, no need for Darby to send them a message when they send him a number.